pkgs/development/libraries/boost/generic.nix

```diff
@@ -149,7 +149,7 @@ stdenv.mkDerivation {
   configureScript = "./bootstrap.sh";
   configureFlags = commonConfigureFlags ++ [
     "--with-icu=${icu}"
-    "--with-python=${python}/bin/python"
+    "--with-python=${python}/bin/python${python.majorVersion}"
```

Comment:
Well, if a symlink would be added, then python2 and python3 would probably collide if installed together. I assume that was the original reason to not create a symlink for python3?